extensions/wikia/Chat2/js/views/views.js

```diff
@@ -580,11 +580,10 @@ var NodeChatUsers = Backbone.View.extend({
 		// admin actions
 		if (actions.admin && actions.admin.length) {
 			var adminActions = ul.clone().addClass('admin-actions');
-			for (var i in actions.admin) { 
-				var action = actions.admin[i],
-					template = _.template( $('#user-action-'+action+'-template').html() );
+			for (var i in actions.admin) {
+				var action = actions.admin[i];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "action" as this name is already used in declaration at line 556. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2814)